src/torchaudio/functional/filtering.py

```diff
@@ -998,7 +998,8 @@ def _lfilter_core(
 
 
 def lfilter(waveform: Tensor, a_coeffs: Tensor, b_coeffs: Tensor, clamp: bool = True, batching: bool = True) -> Tensor:
-    r"""Perform an IIR filter by evaluating difference equation.
+    r"""Perform an IIR filter by evaluating difference equation, using differentiable implementation
+        developed independently by *Yu et al.* :cite:`ismir_YuF23` and *Forgione et al.* :cite:`forgione2021dynonet`.
```

Comment:
```suggestion
    developed independently by *Yu et al.* :cite:`ismir_YuF23` and *Forgione et al.* :cite:`forgione2021dynonet`.
```

I think the extra indentation is causing the first line to become bold for some reason. 

## Current Doc
https://pytorch.org/audio/2.2.0/generated/torchaudio.functional.lfilter.html#torchaudio.functional.lfilter

![Screenshot 2024-01-25 at 3 33 14 PM](https://github.com/pytorch/audio/assets/855818/ae3e728b-bd73-4ffa-9396-7cf9b748ce61)

## Current PR state
https://docs-preview.pytorch.org/pytorch/audio/3734/generated/torchaudio.functional.lfilter.html#torchaudio.functional.lfilter

![Screenshot 2024-01-25 at 3 33 19 PM](https://github.com/pytorch/audio/assets/855818/ab9a2aec-259d-4528-91ee-92e986ed0b5f)

Reply:
Thanks for the catch. Sorry my local computer can't build the doc for some reason so I can't preview the doc immediately. 😅

Reply:
It should work now.
![image](https://github.com/pytorch/audio/assets/17811395/cf774523-da95-44b0-9e6e-43af8d91831e)